libs/data-types/src/lib/enums/MIMEType.ts

```diff
@@ -1,8 +1,9 @@
 import { isEnum } from '@coscrad/validation';
 
-export enum MIMEType {
-    mp3 = 'audio/mpeg',
-    mp4 = 'video/mp4',
-}
+// We may want to exclude this lib from actually knowing the enums
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
```

Comment:
We should probably aim to eliminate all such imports. I think these are pointing to the fact that we need to find a way to 'inject' the additional info into the lib from the applications, keeping the libs ignorant of the specifics. Make this a core value going forward.

Reply:
Tagged this comment with a story.